WordPress/src/main/res/drawable/ic_cloud_illo.xml

```diff
@@ -0,0 +1,24 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We can optimize those long paths as well as remove the `android:fillType`, `android:strokeColor`, and `android:strokeWidth` attributes.  We want to avoid hard-coded colors like `#6F93AD`, which could be `@color/neutral_200` based on similar drawables, but you'll want to check with the designer to make sure.  Also, we should rename `ic_cloud_illo` to something like `img_illustration_cloud_off_152dp` to maintain the naming convention for resources like this one (e.g. `img_illustration_media_105dp`, `img_illustration_pages_104dp`, `img_illustration_posts_75dp`, etc.).  See the code below for the changes.

```xml
<?xml version="1.0" encoding="utf-8"?>

<vector
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:height="102dp"
    android:width="152dp"
    android:viewportHeight="102"
    android:viewportWidth="152">

    <path
        android:fillColor="@color/neutral_200"
        android:pathData="M134,87.3c1.5-0.5,5.3-2,8.1-4.4c2.4-1.9,4.3-4.3,5.6-7s2-5.7,2.2-8.7s-0.1-6.1-0.8-9c-0.9-2.7-1.2-4.4-4.1-8 c-4.7-5.3-8.7-6.9-15.5-8.5c-3-0.5-6.1-0.5-8.9,0.1c-0.6,0.2-1.7-0.3-1.4-1.5l0,0c1.1-3.5,1.4-7.3,0.8-11 c-0.3-4.4-4.4-17.1-21.3-24.7c-7.1-2.9-14.9-3.3-22.5-2.2C72.5,3,68.8,4,65.4,5.7c-1.7,0.9-3.3,1.9-4.8,3.1s-2.8,2.6-4,4.2 c-0.1,0.1-0.2,0.1-0.3,0.1c-0.1-0.1-0.1-0.2-0.1-0.3c2.1-3.4,5.1-6.1,8.6-8.1c3.5-1.9,7.3-3.2,11.3-3.9C78,0.4,81.1,0,84.9,0 C92,0.1,95.8,1.3,99.6,2.7c18.2,7.2,22.2,22.1,22.9,26.2c0.6,3.3,0.3,7-0.4,10.3l0,0c2.6-0.4,5.3-0.2,7.9,0.2 c7.7,1,14.5,6.3,16.8,9.4c2.3,2.6,3.8,6.5,4.4,8.7c0.8,3.2,1.1,6.5,0.8,9.7s-1.1,6.5-2.7,9.4c-1.5,2.9-3.8,5.4-6.4,7.3 c-2.6,1.9-5.7,3.1-8.8,3.8c-0.1,0-0.3-0.1-0.3-0.2C133.8,87.4,133.8,87.3,134,87.3">
    </path>

    <path
        android:fillColor="@color/neutral_200"
        android:pathData="M26.703,0.758C36.249,8.626 45.302,17.018 54.386,25.377L81.642,50.45L108.724,75.706C117.753,84.124 126.814,92.508 135.379,101.416L135.057,101.758C125.51,93.89 116.458,85.497 107.372,77.138L80.116,52.065L53.035,26.809C44.006,18.391 34.946,10.007 26.379,1.1L26.703,0.758Z">
    </path>

    <path
        android:fillColor="@color/neutral_200"
        android:pathData="M104.2,90.8c-7.5,0.7-14.9,0.8-22.4,0.8l-22.4,0.1c-3.7,0-7.5,0-11.2,0c-3.8,0-7.6-0.2-11.3-0.7c-7.5-1-15-3-21.6-7.1 c-3.3-2-6.3-4.5-8.7-7.6c-2.4-3-4.2-6.5-5.3-10.3c-0.5-1.9-0.9-3.8-1.1-5.7C0,58.3,0,56.2,0.1,54.7c0.1-3.9,1.2-7.7,2.9-11.2 c1.7-3.5,3.9-6.6,6.6-9.4c2.6-2.8,5.7-5.1,9.1-6.9c3.4-1.7,7.1-2.9,11-3.1c1.9-0.1,3.8,0,5.7,0.3c1.9,0.3,3.7,0.9,5.4,1.7 c0.1,0.1,0.2,0.2,0.1,0.3c-0.1,0.1-0.2,0.2-0.3,0.1l0,0c-1.7-0.6-3.5-1-5.4-1.3c-1.8-0.2-3.7-0.2-5.5,0c-3.6,0.4-7.1,1.6-10.3,3.4 C13,32.2,7.8,37.8,4.7,44.3c-1.5,3.3-2.5,6.8-2.6,10.4C2,57.3,2,58.5,2.2,60.2c0.2,1.8,0.5,3.6,1,5.3c1,3.5,2.7,6.7,5,9.5 c2.2,2.8,5.1,5.2,8.2,7.1c6.2,3.8,13.5,5.8,20.7,6.7c3.7,0.4,7.3,0.6,11,0.6c3.7,0,7.5,0,11.2,0l22.4,0.1c7.5,0,14.9,0.1,22.4,0.8 c0.1,0,0.2,0.1,0.2,0.3C104.4,90.6,104.3,90.7,104.2,90.8">
    </path>

</vector>
```

Reply:
Thanks @theck13 - in f271bcabe81c68ccb7d6af4812f45674d589913b I updated the drawable and filename. I used `primary_300` as the color (which is currently `#6F93AD`)

Reply:
Thanks @theck13 for giving me a heads up on this one -- we're now using neutral colors for the simple line art illustrations like this one, so `@color/neutral_200` would be appropriate here.

Reply:
Color updated in 00506a921b29f38d3e1dc0ea240343a52f3eb5cf